firmware/thruna/DCM/Src/App/states/App_DriveState.c

```diff
@@ -5,8 +5,11 @@
 #include "states/App_InitState.h"
 #include "App_SetPeriodicCanSignals.h"
 #include "App_SharedConstants.h"
+#include "App_Regen.h"
 
 #define EFFICIENCY_ESTIMATE (0.80f)
+static float apps_pedal_percentage = 0.0f;
+static float prev_torque_request   = 0.0f;
 
 void App_SetPeriodicCanSignals_TorqueRequests()
```

Comment:
can you pass in `apps_pedal_percentage_left/right` here, and fetch `App_CanRx_FSM_PappsMappedPedalPercentage_Get` in `DriveStateRunOnTick100Hz`